pkg/yurthub/proxy/multiplexer/multiplexerproxy.go

```diff
@@ -49,6 +50,10 @@ type multiplexerProxy struct {
 	stop                       <-chan struct{}
 }
 
+func init() {
+	v1.AddToScheme(scheme.Scheme)
```

Comment:
why should add `k8s.io/kubernetes/pkg/apis/core/v1` into scheme?

Reply:
Need to introduce the service conversion function. 
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/v1/conversion.go#L496

Reply:
how about add comments for this init() function and explain why we need to AddToScheme for scheme.Scheme?